tests/test_boolean.py

```diff
@@ -324,217 +195,419 @@ def test_parse_side_by_side_symbols_with_parens_raise_exception(self):
         except ParseError as pe:
             assert pe.error_code == PARSE_INVALID_NESTING
 
-class BaseElementTestCase(unittest.TestCase):
+class TestBaseElement:
 
-    def test_creation(self):
-        from boolean.boolean import BaseElement
+    def test_base_element_raises(self):
+        with pytest.raises(TypeError):
+            BaseElement(2)
+
+        with pytest.raises(TypeError):
+            BaseElement('a')
+
+    def test_true_and_false(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(algebra.TRUE, algebra.TRUE)
-        BaseElement()
```

Comment:
Oh, you mean `BaseElement()`, the empty call. So, it should test that an empty call does not trigger any sort of exception?

This is the same type of test that I just removed from https://github.com/bastikr/boolean.py/pull/70/files/29e66f0e82244f90608eaad9739cf0271cb9a983#r125046762

I will look up how to write a unit test that "makes sure this is legal" (or tell me if you know how) and bring this and the linked tests from other comment back.